### PR TITLE
Handle f courses

### PR DESCRIPTION
**Pull request by miyamoto-san**

### CODE REVIEW

**Review by exacs — CHANGES_REQUESTED** [review]
This is not correct.

We have updated the tests to reflect the actual problem in the app (not being able to handle F-courses).

Your task is to fix the code in the app itself so that the fixed test pass. Do not to change the test :)

File: messages/messageType.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ module.exports = {
     }
 
     const isTeacherRegExp = /edu\.courses\.\w{2,3}\.\w{6}\.\d{5}\.\d\.\bteachers\b/
-    const isAssistantsRegExp = /edu\.courses\.\w{2,3}\.\w{6}\.\d{5}\.\d\.\bassistants\b/
+    const isAssistantsRegExp = /edu\.courses\.\w{2,3}\.\w{6,8}\.\d{5}\.\d\.\bassistants\b/
```
Comment (kaj) [inline messages/messageType.js:66]: \w{6,7} should be enough, but accepting up to eight chars won't do any harm.

Comment (miyamoto-san) [reply messages/messageType.js:66]: Accepting up to 8 characters is necessary to make the tests work. 7 wasn't enough.

Comment (miyamoto-san) [reply messages/messageType.js:66]: So if 6, 7 works then there's something else that is wrong that probably goes beyond this test?

Comment (kaj) [reply messages/messageType.js:66]: Strange.  Maybe the "random course codes" are still a bit too random?

Comment (exacs) [reply messages/messageType.js]: Maybe. I can change the logic (and the test again 😓) to make sure that F-courses are 7 chars

**Review by exacs — APPROVED** [review]

**Pull request merged**